apps/web/scripts/docker/read-secrets.sh

```
@@ -13,4 +13,28 @@ else
   echo "ENCRYPTION_KEY secret not found. Build may fail if this is required."
 fi
 
+if [ -f "/run/secrets/sentry_auth_token" ]; then
+  # Only upload sourcemaps on amd64 platform to avoid duplicate uploads
+  # Sourcemaps are platform-agnostic, so we only need to upload once
+  # TARGETARCH is automatically set by Docker during multi-platform builds
+  if [ "${TARGETARCH:-}" = "amd64" ]; then
+    export SENTRY_AUTH_TOKEN=$(cat /run/secrets/sentry_auth_token)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)